pkg/utils/net.go

```diff
@@ -94,9 +95,19 @@ func WaitForServer(socket string, timeout time.Duration, opts ...interface{}) er
 	}
 }
 
-// ServerActiveAt checks if a gRPC server is accepting connections at the socket.
-func ServerActiveAt(socket string) bool {
-	return WaitForServer(socket, time.Second) == nil
+// IsNonlisteningSocket returns true if nobody is listening at a socket.
```

Comment:
This is probably only me, but I have noticed that I always need to do mental twists with these negative tests when reading the code. Would it be possible to change this to `IsListeningSocket()` instead?